stable/admin/templates/_helpers.tpl

```diff
@@ -1,3 +1,11 @@
+{{/*
+Generic helper function to turn a list into a comma separated string
```

Comment:
I believe this reimplements the helm `join` function:
So you could remove this, and change the statefulset template to simply use:
`{{ .Values.admin.evict.servers | join "," }}`

See http://masterminds.github.io/sprig/ for the list of functions that are included in helm.
(Somewhere there is a list of the Sprig functions that Helm does _not_ include - but I can't find it off-hand...)